internal/tasks/manifests.go

```diff
@@ -733,6 +738,8 @@ func (j *Janitor) doVulnerabilityCheck(account keppel.Account, repo keppel.Repos
 				return fmt.Errorf("Clair reports indexing of %s as finished, but vulnerability report is 404", manifest.Digest)
 			}
 			vulnStatuses = append(vulnStatuses, clairReport.VulnerabilityStatus())
+			now := j.timeNow()
+			vulnInfo.IndexFinishedAt = &now
```

Comment:
This should only be set if the field is nil before, otherwise it will just report when we last checked the vulnerability status.

I was expecting this to show up in a test, but if I'm not mistaken, we don't have a test where an image is already indexed and then the vulnerability status changes in Clair. Please add such a test, or extend an existing test.